build.gradle

```diff
@@ -261,3 +268,13 @@ diffCoverageReport {
         failOnViolation = true
     }
 }
+
+
+tasks.withType(AbstractPublishToMaven) {
```

Comment:
just wondering why does no other plugin need this?

Reply:
This is what I missed in my PR which fixed the publishing. @reta would be a better person to answer this.